client/src/components/EventTypesTab.js

```diff
@@ -210,7 +210,7 @@ export default function EventTypesTab() {
                 </Grid>
                 <Grid item container direction="row" justify="flex-start"
                     alignItems="flex-start">
-                    {cards}
+                    {cards()}
```

Comment:
This was the problem, 
I noticed that if there is no eventType, there is an error in console, so I changed the `map` into a function, and didn't know would cuz this issue.
Just wanna let you so that we could learn a little